NEWS

```diff
@@ -1,3 +1,7 @@
+1.1.0 -- 31 Oct 2015
+
+Added a dependency on htmlwidgets and 'offline' plots are now the default. If you want to create a figure on a plotly server, you need to use `plotly_POST()`. Also added a `config()` function to control the default appearance of the interactive plot
```

Comment:
I just realized that we'll need to change all of our docs to remove the references to the URLs.